lib/ipmi_backend_utils.pm

```diff
@@ -366,6 +368,9 @@ sub set_serial_console_on_vh {
     if (${virt_type} eq "xen" || ${virt_type} eq "kvm") { setup_console_in_grub($ipmi_console, $root_dir, $virt_type); }
     else { die "Host Hypervisor is not xen or kvm"; }
 
+    #Enabling SEV on the machine if it is running SEV/SEV-ES test
+    enable_sev_in_kernel(root_dir => $root_dir) if (get_var('VIRT_SEV_ES_GUEST_INSTALL') and is_x86_64 and ${virt_type} eq "kvm");
```

Comment:
This function is called WITHIN function set_serial_console_on_vh?

Reply:
Yes

Reply:
I mean enabling sev is not related with set_serial_console_on_vh. If editing grub for various different options in one function makes sense , maybe it is needed to update the function name to make the name match what it does?

Reply:
I can update set_serial_console_on_vh to set_grub_on_vh or something else to match its function better.

Reply:
Done